intelmq/bots/experts/abusech/expert_zeus.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+import requests
+from bs4 import BeautifulSoup as bs
+from datetime import datetime as dt
+
+from intelmq.lib.bot import Bot
+from intelmq.lib import utils
+
+
+class AbusechZeuSExpertBot(Bot):
+
+    def process(self):
+        event = self.receive_message()
+        host = event.get("source.fqdn")
+        if not host:
+            host = event.get("source.ip")
+        web_text = requests.get('https://zeustracker.abuse.ch/monitor.php?host=' + host).text
+        web_soup = bs(web_text, 'html.parser')
+        if 'not found' in web_soup.find(attrs={"class": "ContentBox"}).find('p').text:
```

Comment:
lets still use host+' not found ' so that the context is clear
```suggestion
        nf_str = host+' not found '
        if nf_str in web_soup.find(attrs={"class": "ContentBox"}).find('p').text:
```